tor/core/inbox.py

```diff
@@ -160,8 +160,8 @@ def check_inbox(config):
         # In this case, there will be no author attribute.
         if item.author is None:
             send_to_modchat(
-                f'We received a message without an author. Subject: '
-                f'{item.subject}', config
+                f'We received a message without an author -- '
+                f'*{item.subject}*:\n{item.body}', config
```

Comment:
do we want to do two `\n`'s so that we have an empty line here?

Reply:
I based it on `forward_to_slack`, which uses only one newline. I think it looks fine.

Reply:
Roger that, works for me.